command/taskdata_get.go

```diff
@@ -51,6 +51,11 @@ func (c *taskDataGet) Execute(ctx context.Context,
 	if c.File != "" && !filepath.IsAbs(c.File) {
 		c.File = filepath.Join(conf.WorkDir, c.File)
 	}
+
+	if err = createEnclosingDirectoryIfNeeded(c.File); err != nil {
```

Comment:
This differs from the s3 case. Here we create the directory regardless of whether the path is absolute. Should we only create the directory if the path is relative?

Reply:
I've clarified this in the next commit